service-04/PUS-4-2.asn1

```diff
@@ -53,9 +53,6 @@ StatisticsGeneric {UnsignedInteger-Type, ParameterValue-Type, AbsoluteTime-Type,
     standardDeviationValue StandardDeviation-Type OPTIONAL
 }
 
-StandardDeviation ::= CHOICE
-{
-    param1 REAL
-}
+StandardDeviation ::= REAL
```

Comment:
Looks good to me, but the standard says that "standard deviation value field are specific to the parameter identified by the associated parameter ID field". There might have been some problem with `OPTIONAL` and parameters, that's why the latter is missing

Reply:
damn, I missed that. So it should be CHOICE, but parametrized by ID (it was not, that's why I removed it :) )

Reply:
You can try making it parameterizable, maybe I just forgot to do this before, but it can be also the case with OPTIONAL and parameters, I can't remember now.

Reply:
we've switched to V4 in the mean time, so if it compiles it means that OPTIONALs were fixed